src/components/Nav.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavBar } from "../interfaces/NavBar";
+import logo from "../assets/images/logo2.png";
+
+const Nav = () => {
+  const navBarPieces: NavBar[] = [
+    {
+      name: "Főmenü",
+    },
+    {
+      name: "Kedvencek",
+    },
+    {
+      name: "Filmek",
+    },
+    {
+      name: "Sorozatok",
+    },
+  ];
```

Comment:
Egészítsd ki még azzal, hogy path, pl `path:"main-page"`.
Viszont ha az objectet kiegészíted, akkor az interface-t is ki kell, hogy ne csak name legyen neki beállítva.